### ISSUE TITLE
How to restart a timer for an existing task (without creating a new one)?

### ISSUE THREAD

**Issue by levyring** [toggl/toggl_api_docs#96]
I am unable to restart a timer on an existing stopped task using POSTs to https://www.toggl.com/api/v8/time_entries/start

All of my attempts in passing the existing task's "id", with "start" and setting the "duration" to a negative value of current_time + duration(that has already been logged for the task) does not restart the existing task. Instead, a new task with the same description is created. 

When passing the "tid" instead of the "id" for the existing stopped task the API is returning "User cannot access selected task".

Am I missing something?

thanks.

**Comment by theBurlyone** [comment]
Found it, if you hover over a recorded time entry, you will see a play symbol appear next to the time, if you click that it will resume that entry.

**Comment by desrod** [comment]
> I somewhat presume that you mean time entry under 'restart time for an existing task'.
> 
> You can update an existing time entry as needed (in your case - to reflect a running task).
> See - https://github.com/toggl/toggl_api_docs/blob/master/chapters/time_entries.md#update-a-time-entry

What's missing here, is the ability to restart an existing timer, to add new time to it, but _not_ update the time on an existing _stopped_ timer. 

In other words, I've stopped a meeting ("Team Meeting"), and the timer has stopped. Later in the day, I pick up that same meeting and continue it with my team. 

I don't want to create a second "Team Meeting" meeting, with its own, distinct id and timer, I want to resume the previous "Team Meeting", and append the new time to that existing meeting's time, for reporting purposes. 

Since I won't know how long that append will be, I want to restart the timer on the previous meeting, and let it run until the meeting concludes. 

I've tried various endpoint guesses to see if there was a way to resume, continue or create/start a timer by passing in 'id' to the request, to no avail. 

Is it not possible, via the API, to do the same thing I can do in the UI, by clicking the little 'play' button next to a stopped timer?
